pipelined/bevy_wgpu2/src/compute_pass.rs

```diff
@@ -0,0 +1,74 @@
+use crate::{resources::WgpuResourceRefs, WgpuRenderContext};
+use bevy_render2::{
+    pass::ComputePass,
+    pipeline::{BindGroupDescriptorId, ComputePipelineDescriptor, PipelineId},
+    render_resource::BindGroupId,
+    renderer::RenderContext,
+};
+use bevy_utils::tracing::trace;
+
+#[derive(Debug)]
+pub struct WgpuComputePass<'a> {
+    pub compute_pass: wgpu::ComputePass<'a>,
+    pub render_context: &'a WgpuRenderContext,
+    pub wgpu_resources: WgpuResourceRefs<'a>,
+    pub pipeline_descriptor: Option<&'a ComputePipelineDescriptor>,
+}
+
+impl<'a> ComputePass for WgpuComputePass<'a> {
+    fn get_render_context(&self) -> &dyn RenderContext {
+        self.render_context
+    }
+
+    fn set_bind_group(
```

Comment:
Would be nice if we could share this with render pass somehow.. perhaps pull it out into its own function?

Reply:
yeah im down for that!

Reply:
It got a little weird with a `Pass` enum, but I think it works fairly well?

Reply:
I'm guessing that gets optimized out, but it still feels a little weird to me. I actually think id prefer code duplication over that :)

Reply:
I reversed the change.